source/components/reusable/TextField/_tests/TextFieldContainer.test.js

```diff
@@ -1,37 +1,141 @@
 import React from 'react'
 import sinon from 'sinon'
-import { mount } from 'enzyme'
-import { expect } from '../../../../../configuration/testSetup'
+import {
+  mount
+} from 'enzyme'
+import moxios from 'moxios'
+import makeEditable from '../TextFieldContainer'
+import {
+  expect
+} from 'sym/configuration/testSetup'
 import TextFieldContainer from '../TextFieldContainer'
+import globalState from 'sym/source/components/utilities/globalState'
+import { mockGlobalState } from 'sym/source/testUtilities/mockComponentData'
 
-describe( '<TextFieldContainer />', () => {
-  let warnStub, wrapper, makeEditableSpy
+describe('<TextFieldContainer />', () => {
+  let warnStub, wrapper, makeEditableSpy, makeKeyPressProjectSpy, makeKeyPressCouldDoSpy
 
-  beforeEach( () => {
-    warnStub = sinon.stub( console, 'warn' ).callsFake( () => null )
-    makeEditableSpy = sinon.spy( TextFieldContainer.prototype, 'makeEditable' )
-    wrapper = mount( <TextFieldContainer /> )
+  beforeEach(() => {
+    moxios.install()
+    warnStub = sinon.stub(console, 'warn').callsFake(() => null)
+    makeEditableSpy = sinon.spy(TextFieldContainer.prototype, 'makeEditable')
+    makeKeyPressProjectSpy = sinon.spy(globalState, 'updateProjectText')
+    makeKeyPressCouldDoSpy = sinon.spy(globalState, 'updateCouldDoText')
```

Comment:
I think these could be renamed to `updateProjectTextSpy` and `updateCouldDoTextSpy`